iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/facade/ObjectStoreFacade.java

```diff
@@ -0,0 +1,19 @@
+package eu.dnetlib.iis.wf.importer.facade;
+
+/**
+ * ObjectStore service facade.
+ * 
+ * @author mhorst
+ *
+ */
+public interface ObjectStoreFacade {
+
+    /**
+     * Provides all metadata records from given objectstore. 
```

Comment:
True, initialy I've skipped time range parameters and forgot to update comment after introducing `from` and `until` params. Comment updated.